### PR TITLE
Add old bindings for `sign_evm`

### PR DESCRIPTION
**Pull request by DaughterOfMars**
# Description of change

Adds the old nodejs bindings for `sign_evm`.

## Links to any relevant issues

- #505 

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

### CODE REVIEW

File: sdk/src/wallet/bindings/nodejs/lib/Account.ts [hunk 1]
```diff
@@ -42,7 +42,9 @@ import type {
     BuildNftOutputData,
 } from '../types/buildOutputData';
 import type {
+    EvmSignature,
```
Comment (thibault-martinez) [inline sdk/src/wallet/bindings/nodejs/lib/Account.ts]: Where is this defined?

File: sdk/src/wallet/message_interface/message_handler.rs [hunk 2]
```diff
@@ -585,6 +588,20 @@ impl WalletMessageHandler {
                     .await?;
                 Ok(Response::GeneratedEvmAddresses(addresses))
             }
+            AccountMethod::SignEvm { message, chain } => {
+                let msg: Vec<u8> = prefix_hex::decode(message).unwrap();
```
Comment (thibault-martinez) [inline sdk/src/wallet/message_interface/message_handler.rs]: Should probably propagate the error, it's user provided

File: sdk/src/wallet/message_interface/response.rs [hunk 3]
```diff
@@ -99,6 +99,9 @@ pub enum Response {
     /// Response for
     /// [`GenerateEvmAddresses`](crate::wallet::message_interface::AccountMethod::GenerateEvmAddresses)
     GeneratedEvmAddresses(Vec<String>),
+    /// Response for:
+    /// - [`SignEvm`](crate::method::SecretManagerMethod::SignEvm)
+    EvmSignature { public_key: String, signature: String },
```
Comment (thibault-martinez) [inline sdk/src/wallet/message_interface/response.rs:105]: serde camelcase

File: sdk/src/wallet/bindings/nodejs/lib/Account.ts (hunk 1 above)
Comment (DaughterOfMars) [reply sdk/src/wallet/bindings/nodejs/lib/Account.ts]: Oops. 😵‍💫

File: sdk/src/wallet/bindings/nodejs/types/secretManager.ts [hunk 4]
```diff
@@ -54,3 +56,14 @@ export type SecretManager =
     | MnemonicSecretManager
     | StrongholdSecretManager
     | PlaceholderSecretManager;
+
+export interface EvmSignature {
```
Comment (thibault-martinez) [inline sdk/src/wallet/bindings/nodejs/types/secretManager.ts:60]: Can you add it to the .change I added please 🙏🏻 ?

**Review by thibault-martinez — APPROVED** [review]

File: sdk/src/wallet/message_interface/response.rs [hunk 5]
```diff
@@ -99,6 +99,10 @@ pub enum Response {
     /// Response for
     /// [`GenerateEvmAddresses`](crate::wallet::message_interface::AccountMethod::GenerateEvmAddresses)
     GeneratedEvmAddresses(Vec<String>),
+    /// Response for:
+    /// - [`SignEvm`](crate::method::SecretManagerMethod::SignEvm)
```
Comment (qrayven) [inline sdk/src/wallet/message_interface/response.rs:103]:
```suggestion
    /// [`SignEvm`](crate::method::SecretManagerMethod::SignEvm)
```

Comment (qrayven) [reply sdk/src/wallet/message_interface/response.rs:103]: Ok, I see you gave a colon. But other descriptions don't have it so it's up to you.

Comment (DaughterOfMars) [reply sdk/src/wallet/message_interface/response.rs:103]: I copied this from elsewhere, which is why it doesn't fit 🤷

**Review by kwek20 — APPROVED** [review]

File: sdk/src/wallet/message_interface/response.rs (hunk 5 above)
Comment (qrayven) [reply sdk/src/wallet/message_interface/response.rs:103]: ok leave it

**Review by qrayven — APPROVED** [review]

**Pull request merged**